### ISSUE TITLE
ISOBMFF Object Structure clauses

### ISSUE THREAD

**Issue by dwsinger** [MPEGGroup/FileFormat#10]
Subclauses 4.2 and 6.1.2 have the same name and discuss the same aspects. 
We recommend merging 4.2 and 6.1.2.

see also #7

**Comment by dwsinger** [comment]
I disagree, I think 6.1.2 is trying to define the object structure of a presentation (movie, as defined in 3.1.23) and so this can be resolved by renaming 6.1.2 to "Object structure of a presentation"

**Comment by cconcolato** [comment]
That's a good suggestion but there would be some overlap with "4.1 File Structure"

**Comment by dwsinger** [comment]
Chapter 4 is intended to be the basis for files that use box structures. So, it applies to JPEG files that use boxes, and file-type boxes, and the like, but not movie boxes. It applies to HEIF files without a movie box too.

Chapters after 5 apply to timed files/presentations, having a moviebox.

so we have to be careful to maintain this layering

**Comment by dwsinger** [comment]
addressed in 7th edition
